frontend/website/src/Testnet.re

```diff
@@ -211,6 +293,94 @@ module Styles = {
         ),
       ]),
     ]);
+
+  let buttonRow =
+    style([
+      display(`flex),
+      flexWrap(`wrap),
+      marginLeft(`auto),
+      marginRight(`auto),
+      marginTop(rem(3.)),
+      marginBottom(rem(3.)),
+      justifyContent(`spaceAround),
+      alignItems(`center),
+      flexDirection(`column),
+      media(Style.MediaQuery.notMobile, [flexDirection(`row)]),
+    ]);
+
+  let ctaButton =
+    style([
+      padding(`rem(1.125)),
+      background(`rgba((71, 137, 196, 0.1))),
+      border(`px(1), `solid, Style.Colors.hyperlink),
+      borderRadius(`px(6)),
+      maxWidth(`rem(18.75)),
+      marginTop(`rem(0.625)),
+      hover([
+        opacity(0.9),
+        backgroundColor(Style.Colors.azureAlpha(0.2)),
+        border(`px(1), `solid, Style.Colors.hyperlinkHover),
+        cursor(`pointer),
+      ]),
+    ]);
+
+  let ctaContent =
+    style([
+      display(`flex),
+      selector("p", [fontSize(`px(29)), marginTop(`rem(0.4375))]),
+    ]);
+
+  let ctaText = style([marginLeft(`rem(0.625))]);
+
+  let ctaHeading =
+    style([
+      Style.Typeface.ibmplexsans,
+      fontWeight(`num(600)),
+      fontSize(`rem(1.5)),
+      lineHeight(`rem(2.1875)),
+      color(`hex("4782A0")),
+      textAlign(`left),
+    ]);
+
+  let ctaBody =
+    style([
+      Style.Typeface.ibmplexsans,
+      fontStyle(`normal),
+      fontWeight(`normal),
+      fontSize(`px(13)),
+      color(`hex("4782A0")),
+      textAlign(`left),
+      marginTop(`rem(0.3125)),
+    ]);
+
+  let heroRow =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      justifyContent(`spaceAround),
+      alignItems(`center),
+      media("(min-width: 70rem)", [flexDirection(`row)]),
+    ]);
+
+  let heroText =
+    merge([header, style([maxWidth(`px(500)), textAlign(`left)])]);
+
+  let termynal =
+    style([
+      height(`rem(16.875)),
+      marginLeft(`rem(1.875)),
+      marginRight(`rem(1.875)),
+      fontSize(`rem(0.625)),
+      media(
+        Style.MediaQuery.notMobile,
+        [
+          height(`rem(25.)),
+          paddingTop(`rem(4.6875)),
+          paddingLeft(`rem(2.1875)),
```

Comment:
nit: I think generally it's good to only using symmetric padding (if you have something asymmetric better to use margins) due to it composing better. Evidence: https://basscss.com/ , though this link doesn't really explore "why" enough.

Reply:
Also you can use

```
paddingX(`rem(2.1875)) @
paddingY(`rem(4.6875)) @
[ the rest of the syles ]
```

(we have helpers for the symmetric padding)